### PR TITLE
feat: Add commitizen

### PR DESCRIPTION
**Pull request by Galdanwing**
Checking whether everything works by recloning first

### CODE REVIEW

File: backend/HungryBroke/settings/base.py [hunk 1]
```diff
@@ -109,7 +109,6 @@
 
 USE_I18N = True
 
-USE_L10N = True
```
Comment (mehfluffy) [inline backend/HungryBroke/settings/base.py:112]: Why is this deleted? :(

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -33,20 +33,27 @@ repos:
...
 
   - repo: https://github.com/pycqa/flake8
-    rev: 4.0.1
+    rev: 5.0.4
     hooks:
       - id: flake8
         name: flake8-backend
         files: backend/
-        args: ['--config', 'backend/setup.cfg']
+        args: [ '--config', 'backend/setup.cfg' ]
```
Comment (mehfluffy) [inline .pre-commit-config.yaml:41]: can we have [consistency](https://github.com/HungryBroke/HungryBroke/pull/12/files#diff-c0d9756ec1e666a5c4b1ab6fa27d5662156838384a67802e705e4de44d6ab119L9) in the spacing?

File: backend/HungryBroke/settings/base.py (hunk 1 above)
Comment (Galdanwing) [reply backend/HungryBroke/settings/base.py:112]: done by django pre-commit hook
